modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/utils/OpenApiUtil.java

```diff
@@ -0,0 +1,346 @@
+                            "message",
+                            "success"
+                          ],
+                          "type": "object",
+                          "properties": {
+                            "code": {
+                              "type": "string",
+                              "description": "The result code of response."
+                            },
+                            "message": {
+                              "type": "string",
+                              "description": "The result message of response."
+                            },
+                            "success": {
+                              "type": "boolean",
+                              "description": "The success boolean of response."
+                            }
+                          }
+                        },
+                        "property": {
+                          "required": %s,
+                          "type": "object",
+                          "properties": %s
+                        },
+                        "CreateRequest": {
+                          "required": [
+                            "category",
+                            "csp",
+                            "flavor",
+                            "name",
+                            "region",
+                            "version"
+                          ],
+                          "type": "object",
+                          "properties": {
+                            "category": {
+                              "type": "string",
+                              "description": "The category of the service",
+                              "enum": [
+                                "ai",
+                                "compute",
+                                "container",
+                                "storage",
+                                "network",
+                                "database",
+                                "media_service",
+                                "security",
+                                "middleware",
+                                "others"
+                              ]
+                            },
+                            "name": {
+                              "type": "string",
+                              "description": "The name of the service"
+                            },
+                            "version": {
+                              "type": "string",
+                              "description": "The version of service"
+                            },
+                            "region": {
+                              "type": "string",
+                              "description": "The region of the provider."
+                            },
+                            "csp": {
+                              "type": "string",
+                              "description": "The csp of the Service.",
+                              "enum": [
```

Comment:
can this list be also read dynamically?

Reply:
Yes. I will get the list enum of Csp and fill the list here.